packages/processors/src/helpers/pricing.ts

```diff
@@ -0,0 +1,71 @@
+import { IPricingProvider } from "@grants-stack-indexer/pricing";
+import { Token } from "@grants-stack-indexer/shared";
+
+import { TokenPriceNotFoundError } from "../internal.js";
+import { calculateAmountInToken, calculateAmountInUsd } from "./tokenMath.js";
+
+// sometimes coingecko returns no prices for 1 hour range, 2 hours works better
+const TIMESTAMP_DELTA_RANGE = 2 * 60 * 60 * 1000;
```

Comment:
there isn't a specific api behaviour here actually, this comment is smth Gitcoin team found out experimenting,
i think for now is not needed an env variable but now im thinking if maybe this can be the default value on Coingecko provider instead of API caller passing `timestamp.now() + DELTA_RANGE` everytime. i leave it for reviewing later 🫡

Reply:
agree with you nigiri, lets have this value as default ( within the CoingeckoProvider) and make the param optional , wdyt ?

Reply:
will add a task in linear to refactor in another PR

Reply:
https://linear.app/defi-wonderland/issue/GIT-144/refactor-pricingprovider-coingecko-to-make-optional-the-to-timestamp